### DISCUSSION TITLE
Possible updater problem

### DISCUSSION THREAD

**Discussion by WayCon2** [dosemu2/dosemu2#2285] [General]
Some months ago the dosemu2 updates were not functioning properly and were soon repaired. I believe that is happening again. Here is what I get in terminal: 

(base) wayne@OPlex-7050:~/.dosemu/drive_c$ dosemu
ERROR: MFS: /home/wayne/.dosemu/drive_c/ unaccessible
ERROR: INT21: redirecting C: failed (err = 3)
(base) wayne@OPlex-7050:~/.dosemu/drive_c$

I await any possible response. Thank you,

WayCon2

**Comment by stsp** [comment]
Moved to #2286

**Comment by WayCon2** [comment]
Thank you.wc

    On Monday, October 7, 2024 at 10:16:10 AM EDT, Stas Sergeev ***@***.***> wrote:  
 
 


Moved to #2286
